src/Arcus.Security.Tests.Unit/KeyVault/Stubs/KeyVaultClientFactoryStub.cs

```diff
@@ -1,13 +1,13 @@
 ﻿using System;
 using System.Threading.Tasks;
-using Arcus.Security.Providers.AzureKeyVault.Factories;
+using Arcus.Security.Providers.AzureKeyVault.Authentication.Interfaces;
 using Microsoft.Azure.KeyVault;
 
 namespace Arcus.Security.Tests.Unit.KeyVault.Stubs
 {
-    internal class KeyVaultClientFactoryStub : KeyVaultClientFactory
+    internal class KeyVaultClientFactoryStub : IKeyVaultAuthenticator
```

Comment:
😊 this is maybe nitpicking: this isn't exactly a _Stub_ because it's not implemented. If it's just to just have an instance, it's a _Dummy_ or _Pseudo_ _Test Double_.

Reply:
That's try, feel free to PR a suggestion!

Reply:
Changed it to dummy.